lib/playlist-generation/themes/index.ts

```diff
@@ -0,0 +1,23 @@
+import { AudioFeatures } from '../..';
+import { dissociating } from './dissociating';
+
+export type ThemeConditions = {
+  min: number,
+  max: number,
+  target: number,
+  weight: number,
+}
+
+export const NO_WEIGHT: ThemeConditions = { min: 0, max: 0, target: 0, weight: 0 };
```

Comment:
won't NO_WEIGHT reject all songs because the score won't be > and < 0?  based on computeScore

Reply:
thats the whole point of it right?

Reply:
o wait no i see what ur saying nice catch